File: EventQRBackend/eventqr/src/main/java/com/thedavelopers/eventqr/features/uploads/model/dto/StoredFileResponse.java
package com.thedavelopers.eventqr.features.uploads.model.dto;

import java.time.Instant;
import java.util.UUID;

public record StoredFileResponse(UUID fileId, UUID ownerId, String purpose, String fileName,
                                 String contentType, long size, String status, Instant storedAt) {
}

File: EventQRMobile/app/src/main/java/com/thedavelopers/eventqr/features/uploads/model/dto/StoredFileResponse.kt
package com.thedavelopers.eventqr.features.uploads.model.dto

import java.time.Instant
import java.util.UUID

data class StoredFileResponse(
    val fileId: UUID,
    val ownerId: UUID? = null,
    val purpose: String? = null,
    val fileName: String? = null,
    val contentType: String? = null,
    val size: Long = 0,
    val status: String? = null,
    val storedAt: Instant? = null,
    val contentBase64: String? = null,
)